### PR TITLE
feat: add new env public node url

### PR DESCRIPTION
**Pull request by luizstacio**
Add a new env config to set the public node URL. This is used to correctly inform the node URL on the faucet page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ The faucet makes use of environment variables for configuration.
 | CAPTCHA_SECRET       | The secret key used for enabling Google captcha authentication.         |
 | WALLET_SECRET_KEY    | A hex formatted string of the wallet private key that owns some tokens. |
 | FUEL_NODE_URL        | The GraphQL endpoint for connecting to fuel-core.                       |
-| SERVICE_PORT         | The port the service will listen for http connections on.               |
+| PUBLIC_FUEL_NODE_URL | The public GraphQL endpoint for connecting to fuel-core. Ex.: https://node.fuel.network/graphql |
+| PORT         | The port the service will listen for http connections on.               |
```
Comment (adlerjohn) [inline README.md:20]: Shouldn't this still be `SERVICE_PORT`?
```suggestion
| SERVICE_PORT         | The port the service will listen for http connections on.               |
```

Comment (luizstacio) [reply README.md:20]: Here: https://github.com/FuelLabs/faucet/blob/8ec49d945b23d080dea636ff3f51610b5c3a432a/src/constants.rs#L14 you can see that to change the `PORT` the name of the env was change to just `PORT`.

Comment (adlerjohn) [reply README.md:20]: Ah I see. At the risk of being anal, can you split this change into a separate PR? It's really not obvious that it's an orthogonal fix!

Comment (luizstacio) [reply README.md]: @adlerjohn I remove the change

**Review by adlerjohn — COMMENTED** [review]
it looks fine to me, but would like @Voxelot to confirm

**Review by Voxelot — APPROVED** [review]

**Pull request merged**